mkdocs.yml

```diff
@@ -1,8 +1,8 @@
 site_name: Fiware-Orion
 site_url: https://fiware-orion.readthedocs.org
-repo_url: https://github.com/telefonicaid/fiware-orion.git
+repo_url: https://github.com/telefonicaid/fiware-orion
 site_description: Orion Context Broker Documentation
-docs_dir: doc/manuals
+docs_dir: doc
```

Comment:
I'm not sure about this change... As far as I understand `docs_dir` is the base path for directories in the repository. So, for instance, `- 'Introduction': 'user/index.md'`  refers to `docs_dir` value + `'user/index.md'`. In this sense, I think `doc/manuals` is ok. If this were not right, then https://fiware-orion.readthedocs.org/en/master/ will not work at all :)

What do you think?